CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ Here is an overview of all new **experimental** features:
 
 - **General**: Add a warning when KEDA run outside supported k8s versions ([#4130](https://github.com/kedacore/keda/issues/4130))
 - **General**: Use (self-signed) certificates for all the communications (internals and externals) ([#3931](https://github.com/kedacore/keda/issues/3931))
+- **General**: Improve test coverage in `pkg/util` ([#3871](https://github.com/kedacore/keda/issues/3871))
```

Comment:
just a nit, please move this to `### Other` sections as it is not user facing feature

Reply:
and pls don't forget to sort alphabetically

Reply:
Thanks for the review. I have move that line to `Other` and did some minor formatting too :pray: